roles/splunk_common/tasks/main.yml

```diff
@@ -130,4 +131,4 @@
 - include_tasks: add_splunk_license.yml
 
 - include_tasks: disable_popups.yml
-  when: "'disable_popups' in splunk and splunk.disable_popups | bool"
\ No newline at end of file
+  when: "'disable_popups' in splunk and splunk.disable_popups | bool"
```

Comment:
Was this change intentional?

Reply:
nope I don't think so I made any change in that.

Reply:
Looks like a few extra spaces might have been removed on line 135 and that is fine. Closing the comment thread now.